package.json

```diff
@@ -41,6 +41,7 @@
 		"clsx": "2.1.1",
 		"cmdk": "0.2.1",
 		"commerce-kit": "0.0.11",
+		"lodash": "4.17.21",
```

Comment:
lodash/pick -> https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_pick